railties/test/generators/plugin_generator_test.rb

```diff
@@ -201,6 +228,12 @@ def test_action_mailer_is_removed_from_frameworks_if_skip_action_mailer_is_given
     assert_file "test/dummy/config/environments/production.rb" do |content|
       assert_no_match(/config\.action_mailer/, content)
     end
+    assert_no_directory "test/dummy/app/mailers"
+  end
+
+  def test_action_cable_is_removed_from_frameworks_if_skip_action_cable_is_given
+    run_generator [destination_root, "--skip-action-cable"]
+    assert_file "test/dummy/config/application.rb", /#\s+require\s+["']action_cable\/engine["']/
```

Comment:
No, the test fails.
Only
`assert_file "test/dummy/config/application.rb", /#\s+require\s+["']action_cable\/engine["']/` it looks strange